vizro-core/tests/unit/vizro/models/test_dashboard.py

```diff
@@ -102,6 +107,64 @@ def test_page_registry_with_title(self, vizro_app, page_1, mocker):
         mock_register_page.assert_any_call(
             module=page_1.id,
             name="Page 1",
+            description="",
+            image=None,
+            title="My dashboard: Page 1",
+            path="/",
+            order=0,
+            layout=mocker.ANY,  # partial call is tricky to mock out so we ignore it.
+        )
+
+    def test_page_registry_with_description(self, vizro_app, mocker):
+        mock_register_page = mocker.patch("dash.register_page", autospec=True)
+        vm.Dashboard(
+            pages=[vm.Page(title="Page 1", components=[vm.Button()], description="My description")]
+        ).pre_build()
+
+        mock_register_page.assert_any_call(
+            module="Page 1",
+            name="Page 1",
+            description="My description",
+            image=None,
+            title="Page 1",
+            path="/",
+            order=0,
+            layout=mocker.ANY,  # partial call is tricky to mock out so we ignore it.
+        )
+
+    @pytest.mark.parametrize(
+        "image_filename", ["app.png", "app.svg", "folder/app.png", "folder/app.svg", "logo.png", "logo.svg"]
+    )
+    def test_page_registry_with_image(self, page_1, mocker, tmp_path, image_filename):
+        Path(tmp_path / "folder").mkdir()
```

Comment:
```suggestion
        if image_path.parent != Path("."):
            Path(tmp_path / image_path.parent).mkdir()
```
To avoid (a) hardcoding the subdirectory name and (b) creating the directory for test cases that don't need it.